lib/hz2600/CallflowBuilder/Builder.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace CallflowBuilder;
+
+use CallflowBuilder\Node\AbstractNode; 
+
+class Builder {
```

Comment:
Fixed by adding a fromBuilder method to the API Entity callflow. Now this just inherits the whole data payload from the Builder allowing us to assign the phonenumbers/pattens via the builder.